x-pack/plugins/apm/public/plugin.ts

```diff
@@ -100,11 +119,11 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
         const { renderApp } = await import('./application');
         // Get start services
         const [coreStart] = await core.getStartServices();
+        await setupLazyStuff(coreStart);
 
-        // render APM feedback link in global help menu
-        setHelpExtension(coreStart);
-        setReadonlyBadge(coreStart);
-
+        const { createStaticIndexPattern } = await import(
+          './application/dynamicImports'
+        );
```

Comment:
This could be parallelised better, and I'm not sure why we have a separate file for dynamic imports if we are also loading stuff asynchronously from `./application`

Reply:
improved